src/StackNavigation.Uno.WinUI/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Chinook.SectionsNavigation.Uno.WinUI")]
```

Comment:
This AssemblyInfo only mention itself (StackNaviation.Uno.WinUI), was that a fix for an internal accessibility issue or anything else? I think only adding WinUI to the abstractions one should be enough

Reply:
Oh nvm I read that badly, disregard this